### PR TITLE
Workflow annotation

### PR DESCRIPTION
**Pull request by bedroesb**
To have a better link between workflow and tutorial, I placed the title of the tutorial in the annotation section of the workflow. This also serves as some extra information in many situations (helpful if people just scroll trough the workflows on a galaxy instance). This can be a confirmation that the workflow belongs to a specific tutorial.

### CODE REVIEW

File: topics/ecology/tutorials/ref-based-rad-seq/workflows/Galaxy-Workflow-Workflow_constructed_from_history__STACKS_RAD__population_genomics_with_reference_genome_.ga [hunk 1]
```diff
@@ -1002,7 +1002,7 @@
           "type": "tabular"
         },
         {
-          "name": "out_fastpĥase",
+          "name": "out_fastp\u0125ase",
```
Comment (bgruening) [inline topics/ecology/tutorials/ref-based-rad-seq/workflows/Galaxy-Workflow-Workflow_constructed_from_history__STACKS_RAD__population_genomics_with_reference_genome_.ga]: uha, what is this ....

File: topics/ecology/tutorials/ref-based-rad-seq/workflows/workflow.ga [hunk 2]
```diff
@@ -624,7 +624,7 @@
           "type": "tabular"
         },
         {
-          "name": "out_fastpĥase",
+          "name": "out_fastp\u0125ase",
```
Comment (bgruening) [inline topics/ecology/tutorials/ref-based-rad-seq/workflows/workflow.ga]: same here ...

**Comment by bedroesb** [comment]
Nice catch, the non ascii characters are wrongly dumped by the json library in python. @hexylena already explained me that these are linked to the version of this tool. Maybe it would be a good idea to change to the newer version of this tool solving this?

**Comment by bgruening** [comment]
Tool update sounds nice too me ... we just to ensure the tutorial works. But I guess we can ask @yvanlebras for help.

**Comment by hexylena** [comment]
Do you want to update your linter to check for this as well? It's a nice feature and would be good if future workflows conformed.

File: bin/validate-workflow-tags.sh [hunk 3]
```diff
@@ -31,7 +34,7 @@ do
                 echo "Checking tutorial $w for tags" 
                 if tester $w $topic;
                 then
-                    echo "ERROR: No 'tags' attribute found with its corresponding topic or workflow includes tool(s) from testtoolshed."
+                    echo "ERROR: No 'tags' attribute found with its corresponding topic or 'annotation' attribute with for example the title of the tutorial or workflow includes tool(s) from testtoolshed."
```
Comment (hexylena) [inline bin/validate-workflow-tags.sh]: you might consider:

1. in the tester script write to sys.stderr with the message that's more clear/detailed ("in wf X an annotation is missing, it should be Y")
2. Not writing an error in the bash portion and just setting the overall exit code

Comment (bedroesb) [reply bin/validate-workflow-tags.sh]: don't know 100% what you mean with  just setting the overall exit code, but I've added sys.stderr codes. Thanks for the suggestion.

**Comment by bedroesb** [comment]
I included a mistake in one of the workflows on purpose to test the linting

File: bin/validate-workflow-tags.sh (hunk 3 above)
Comment (hexylena) [reply bin/validate-workflow-tags.sh]: The changes you made look good :)

**Comment by bedroesb** [comment]
I think this is good to go :+1:

**Comment by hexylena** [comment]
Looks great! Thanks for the linting too

**Pull request merged**